enterprise/backend/src/metabase_enterprise/metabot_v3/tools/util.clj

```diff
@@ -22,10 +22,12 @@
   [column]
   (let [column (u/normalize-map column)]
     (cond
-      (lib.types.isa/boolean? column)               "boolean"
-      (lib.types.isa/string-or-string-like? column) "string"
-      (lib.types.isa/numeric? column)               "number"
-      (lib.types.isa/temporal? column)              "date")))
+      (lib.types.isa/boolean? column)                "boolean"
+      (lib.types.isa/string-or-string-like? column)  "string"
+      (lib.types.isa/numeric? column)                "number"
+      (isa? (:effective-type column) :type/DateTime) "datetime"
+      (isa? (:effective-type column) :type/Time)     "time"
```

Comment:
Nice. 
NOTE: Without changing ai-service this will likely break the integration (type checks) so we need to wait on this to be merged.
I created a new issue for tackling ai-service: https://linear.app/metabase/project/metabot-embedding-use-case-012c7f36ce09/issues

Reply:
@Somtom, please let me know when this can be merged.

Reply:
@metamben Go for it: https://github.com/metabase/ai-service/pull/161 is merged